daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1617,28 +1649,9 @@ private[lf] object SBuiltin {
               }
             }: Option[V.ContractId] => (Control, Boolean)
 
-            // TODO (drsk) validate key hash. https://github.com/digital-asset/daml/issues/13897
```

Comment:
As disclosure (contract key) table is built using known good data, there is no longer any need to validate disclosed contract key hashes